Dockerfile

```diff
@@ -21,18 +24,16 @@ ADD . .
 
 # Pass a Git short SHA as build information to be used for displaying version
 RUN SHORT_SHA=$(git rev-parse --short=12 HEAD); \
-    go build \
+    packr build \
```

Comment:
I'm worried about this one a bit. Is it possible to generate files, commit and `go build` without `packr`?
This will prevent GoReleaser from compiling assets to the binary, I believe.

Reply:
I'm not sure, it's my first time using packr but I'll check. Why should we prevent GoReleaser from compiling assets to the binary?

Reply:
Because it will call plain `go build`, not `packr build`. We can probably override that, but I would prefer not to, if we can.

Reply:
Seems like V2 solves this exact problem, I'll update the code to use it.